### DISCUSSION TITLE
Where can we find LUA SCRIPT?

### DISCUSSION THREAD

**Discussion by samsab1** [FrSkyRC/ETHOS-Feedback-Community#4653] [Q&A]
hello everyone
sorry for my crude question, but I do not master the concept of LUA at all. however I sometimes see Frsky radio screens, like the X20S that I have, dressed by a global script allowing to have all the necessary information including a large part related to telemetry. this differs from the classic segmented screen in 9 parts but which does not allow to have all the information presented in a more elegant and dynamic way.

So I ask the question: where can I find these scripts, if possible dedicated to Quadcopter + INAV combinations, Flying wing + INAV, Classic 3-axis aircraft + INAV
thank you for your answers

**Comment by robthomson** [comment]
Quite often these end up being sourced either from various user groups in the area (discord or fb groups)

Inav currently does not have much in the way of lua for ethos.   Likewise betaflight is similar.

But it's slowly changing.

I guess a question really is.. What are you wanting exactly?

**Comment by samsab1** [comment]
In principle something like this: a single screen where all the essential information is displayed in a less Stalinist way, that is to say in an elegant, organized and readable way.
![2024-10-23_11h45_34](https://github.com/user-attachments/assets/6a4ea64c-9565-41a2-8283-7d71f1e73ce6)
https://www.youtube.com/watch?v=UqotF4I24Q4

**Comment by Benno00** [comment]
Check out Yaapu
https://github.com/yaapu/FrskyTelemetryScript
![image](https://github.com/user-attachments/assets/b59ace40-f660-4d24-b514-6eb1262d71ad)
